src/solvers/strings/string_constraint_generator_format.cpp

```diff
@@ -267,61 +268,42 @@ get_component_in_struct(const struct_exprt &expr, irep_idt component_name)
 /// \return String expression representing the output of String.format.
 static std::pair<array_string_exprt, string_constraintst>
 add_axioms_for_format_specifier(
-  symbol_generatort &fresh_symbol,
-  const format_specifiert &fs,
-  const struct_exprt &arg,
-  const typet &index_type,
-  const typet &char_type,
-  array_poolt &array_pool,
-  const messaget &message,
-  const namespacet &ns)
-{
+    symbol_generatort &fresh_symbol, const format_specifiert &fs,
+    const std::function<exprt(const irep_idt &)> &get_arg,
+    const typet &index_type, const typet &char_type, array_poolt &array_pool,
+    const messaget &message, const namespacet &ns) {
```

Comment:
Why all the weird reformatting?

Reply:
my clang-format is broken :sob:

Reply:
Did #4461 possibly trip it up as you may need to move to a newer version?

Reply:
Oh yes that looks related: 
```
.git-rewrite/t/.clang-format: Invalid argument
YAML:63:21: error: unknown key 'IndentPPDirectives'
IndentPPDirectives: AfterHash
```
I'm using clang-format-6.0

Reply:
That seems surprising, it's been added in `clang-format` 6 (and `clang-format-6.0` works for me). But you might also try upgrading to `clang-format` 7, which is what we use on Travis.

Reply:
it's not available on ubuntu 16.04

Reply:
http://apt.llvm.org/ has repositories for Xenial.